### ISSUE TITLE
package.json fields references wrong files

### ISSUE THREAD

**Issue by formatlos** [alexkhismatulin/react-use-count-down#47]
The published version has flattened file structure, which means the references to `source`, `main`, `module`, `unpkg` in package.json are wrong, leading to problems with bundlers using those fields (e.g. vite).

v1.1.3
![v1 1 3](https://user-images.githubusercontent.com/214001/233397307-999fe929-4572-48a8-a461-3e93012eab61.png)

v1.1.1
![v1 1 1](https://user-images.githubusercontent.com/214001/233397313-e4b19c13-a937-450f-aee7-2f332bae6e73.png)

**Comment by frzkn** [comment]
Yes, this broke the bundler when the dependency got auto-resolved to `v1.1.3`
Downgrading fixes it.

**Comment by HuyAms** [comment]
I got the same issue when running test with vitest. 

Error: Failed to resolve entry for package "react-countdown-hook". The package may have incorrect main/module/exports specified in its package.json.

**Comment by LesTontonsDev** [comment]
@alexkhismatulin Can you please make a new correct release?
